tests/test_filtering.py

```diff
@@ -36,7 +35,7 @@ def test_urgent(tmpdir, runner, create):
     create(
         'one.ics',
         'SUMMARY:haha\n'
-        'PRIORITY: 9\n'
+        'URGENT: 9\n'
```

Comment:
>If priority = 9 implies urgent

Yeah that part confuses me as well. I think the tests have a bug. The iCalendar RFC clearly states that 1-4 is priority=high